src/operation/get-parameters-as-json-schema.ts

```diff
@@ -189,118 +203,146 @@ export default function getParametersAsJsonSchema(operation: Operation, api: OAS
   function transformParameters(): SchemaWrapper[] {
     const operationParams = operation.getParameters();
 
-    return Object.keys(types).map(type => {
```

Comment:
This is all whitespace because I changed it to being set to a `transformed` constant. View this diff with whitespace changes hidden.